tools/clang/unittests/HLSLExec/LongVectors.cpp

```diff
@@ -12,9 +12,11 @@
 
 #include "HlslExecTestUtils.h"
 
+#include <algorithm> // For sort
 #include <array>
 #include <bitset>
 #include <iomanip>
+#include <numeric> // For accumulate
 #include <optional>
```

Comment:
```suggestion
#include <algorithm>
#include <array>
#include <bitset>
#include <iomanip>
#include <numeric>
#include <optional>
```

These "for foo" comments get stale really quickly and I don't think add any actual value.